requirements.txt

```diff
@@ -1 +1 @@
-pipenv == 2023.11.15 --hash=sha256:6eb537da5a27671f71dc596ea96a5b341aae9a1695413beeb95364467409bb3d
+pipenv == 2023.11.15
```

Comment:
this was intentionally locked to specific hash and it should be updated separately if needed

Reply:
`pipenv == 2023.11.15` locks to the exact version perfectly fine. The only thing it does not do is to validate the integrity of the version, but I don't think we need that here. Using `--hash` and not specifying hashes of all the dependencies in the `requrements.txt` forces you user to use `--no-deps`, which is wrong. If the user does not have all the dependencies preinstalled they should be allowed to simply use run `pip install -r requrements.txt`.